CMakeLists.txt

```diff
@@ -274,7 +274,7 @@ set(ROS_VERSION "kinetic" CACHE STRING "The ROS distribution version.")
 if (BUILD_ROS_PLUGINS)
   set(CMAKE_PREFIX_PATH ${CMAKE_PREFIX_PATH} /opt/ros/${ROS_VERSION}/)
   find_package(catkin REQUIRED COMPONENTS roscpp sensor_msgs std_msgs
-    geometry_msgs nav_msgs tf image_transport cv_bridge tf2_ros tf2)
+    geometry_msgs nav_msgs tf image_transport cv_bridge tf2_ros tf2 tf2_geometry_msgs mavros_msgs)
```

Comment:
Can you move this reference to mavros_msgs to the ROSAirSim CMakeLists.txt? That way to build general ROS plugins, they don't need to install mavros_msgs as well. However, they will have to install mavros_msgs to use AirSim, I think this is fine.

Reply:
Hey Kevin I changed this, but if failed on the 16.04 build because "rosdep upate" couldn't run. Maybe the bionic key server was down? Could you try re-running the travis build on 16.04?